### PR TITLE
change intel logo

### PR DESCRIPTION
**Pull request by andesol**
## Done

- Change logo according to the copy doc
- Update link

## QA

- Visit demo: https://ubuntu-com-12432.demos.haus/internet-of-things
- See Felicia's comment in the [copy doc](https://docs.google.com/document/d/1nBx1SP3Y10gXX0rZPDaz1Gudy-hWP4XRMUcETyPBldI/edit#heading=h.iqtg6b1bg6rk)
- Check that the logo is correct

## Issue / Card

Fixes https://github.com/canonical/workplace-engineering-squad/issues/872

### CODE REVIEW

File: templates/internet-of-things/index.html [hunk 1]
```diff
@@ -416,7 +416,7 @@ <h2>From prototype to production on all major platforms</h2>
       <header class="no-border u-align--center u-vertically-center u-hide--medium u-hide--small" style="min-height: 7rem;">
         {{
           image(
-            url="https://assets.ubuntu.com/v1/8cb47950-g46.png",
+            url="https://assets.ubuntu.com/v1/efca8490-intel-logo-0.png",
             alt="",
             width="90",
             height="65",
```
Comment (petesfrench) [inline templates/internet-of-things/index.html:422]:
```suggestion
            height="90",
```

**Review by petesfrench — CHANGES_REQUESTED** [review]
Can you update the size of this image please, it needs to be a square

**Comment by petesfrench** [comment]
@andesol There is also a link that needed updating, I have tagged you in the copydoc where

**Comment by andesol** [comment]
Thanks @petesfrench! Should be ok now

**Review by petesfrench — APPROVED** [review]
LGTM

**Pull request merged**